pkg-scripts/postinstall.sh

```diff
@@ -5,6 +5,8 @@ CONFIG_PATH=/etc/cagent/cagent.conf
 if [ "$1" = configure ]; then
     # $2 contains previous version number
     if [ -z "$2" ]; then # fresh install
+        chown cagent:cagent -R /var/lib/cagent
+        chmod 6777 /var/lib/cagent/jobmon
```

Comment:
postinstall will run before the first run of jobmon, so `/var/lib/cagent/jobmon` may not exists? Should you `mkdir -p` it?

Reply:
the folder will be created by package manager (DEB or RPM) before running post install script. The folders to be created are listed in goreleaser config file.

Reply:
what if the user uses the binary-only release(we still do it). Will `cagent` just fail because of the missing directory?

Reply:
nope, jobmon creates the directory if it does not exists.

Adding the directory to package-manager instructions allows to remove it automatically when removing package (with purge option).

Reply:
got it 👍